### PR TITLE
Translate release 2.0.0-p643 and EOL 1.9.3 announcements (es)

### PR DESCRIPTION
**Pull request by dabit**
@vic @FerPerales

### CODE REVIEW

File: es/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+A partir de hoy, todo el soporte para Ruby 1.9.3 ha terminado. Las correcciones
+de seguridad y funcionalidad aplicados en versiones más nuevas de Ruby ya no serán
+aplicadas en 1.9.3.
+
+El fin de este ciclo fue anunciado [hace casi un año](https://www.ruby-lang.org/es/news/2014/01/10/ruby-1-9-3-will-end-on-2015/).
+
+Recomendamos que actualices a una versión de Ruby 2.0.0 o posterior lo antes
+posible. Por favor contáctanos si quieres seguir manteniendo la versión 1.9.3
+o si por algunarazón no puedes actualizar tu versión de Ruby.
```
Comment (FerPerales) [inline es/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: alguna razón (missing whitespace)

File: es/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+layout: news_post
+title: "El soporte para Ruby 1.9.3 ha terminado"
+author: "Olivier Lacan"
+translator: "David Padilla"
+date: 2015-02-23 00:00:00 +0000
+lang: es
+---
+
+A partir de hoy, todo el soporte para Ruby 1.9.3 ha terminado. Las correcciones
+de seguridad y funcionalidad aplicados en versiones más nuevas de Ruby ya no serán
```
Comment (FerPerales) [inline es/news/_posts/2015-02-23-support-for-ruby-1-9-3-has-ended.md]: Las correciones ... __aplicadas__

**Comment by FerPerales** [comment]
@dabit @vic there are a couple of suggestions, but PR has been already merge :(

**Comment by vic** [comment]
@FerPerales  I merged the PR and just added the whitespace afterwards. ^^ 

https://github.com/ruby/www.ruby-lang.org/commit/be500f54e51325b914e84fbb2600e854a744fd4b

**Comment by dabit** [comment]
Thanks to both!

**Pull request merged**